infrastructure/sysdig/sysdig-team.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: 61d198-sysdigteam
+objects:
+  - apiVersion: ops.gov.bc.ca/v1alpha1
+    kind: SysdigTeam
+    metadata:
+      name: 61d198-sysdigteam
+    spec:
+      team:
+        description: bcparks.ca sysdig team
+        users:
+          - email: hi@cole.io
+            role: ROLE_TEAM_MANAGER
+          - email: neal.li@gov.bc.ca
+            role: ROLE_TEAM_MANAGER
+          - email: day@oxd.com
+            role: ROLE_TEAM_EDIT
+          - email: mark@digitalspace.ca
+            role: ROLE_TEAM_EDIT
+          - email: cameron@digitalspace.ca
+            role: ROLE_TEAM_EDIT
+          - email: goldie.zohar@gmail.com
```

Comment:
Oh actually... this should be `name` (not `email`) and role is required for all users

Reply:
Fixed in https://github.com/bcgov/bcparks.ca/pull/236/commits/01976f290daab0c505959a8c93730f4bff43c190